code-gen-projects/rust/code-gen-demo/build.rs

```diff
@@ -0,0 +1,36 @@
+// build.rs
+
+use std::env;
+use std::io;
+use std::io::*;
+
+fn main() {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+    let out_dir = env::var("OUT_DIR").unwrap();
+
+    // Invoke cargo CLI
+    let ion_cli = "../../../target/debug/ion";
```

Comment:
You should pass in the path from the integration tests.

1. This is not something that customers can copy. Getting it from the environment variable or the user path is something that customers can copy.
2. Hard coded paths like this are brittle.
3. This test project should not have to know anything about the outer project that contains it.

Reply:
Thanks for the explanation, I have modified according to your suggestions in a Follow-up PR: https://github.com/amazon-ion/ion-cli/pull/98